spec/factories/push_notification_subscriptions.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :push_notification_subscription do
```

Comment:
The factory is not very useful by itself. In my opinion, it should provide minimal data for the record to be valid.
Or you can skip creating the factory and just create `PushNotificaitonSubscription` in the test directly.

Reply:
I went for the 2nd option.